rest/handler.go

```diff
@@ -852,3 +852,14 @@ func getRestrictedInt(rawValue *uint64, defaultValue, minValue, maxValue uint64,
 
 	return value
 }
+
+func (h *handler) getStatPrefix() (string) {
```

Comment:
This is too expensive, and the db name lookup will need to be done when appropriate by callers anyway.  Should be removed and handled by moving the expvar call post-db lookup, and invoking on the DatabaseContext.

Reply:
Yeah, I wondered if this might be a bit heavy. The reasoning behind having it here is to catch all incoming calls in one place, otherwise we'd have to be handling the stats in each individual `handle*()`, right? I'm not sure how we'd catch things non-existant db names for that (which we already do for the rest stats and top level `requests_total` type stats).